<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarvoorsGUI</class>
 <widget class="QWidget" name="StarvoorsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StaRVOOrS</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>341</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_main">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QTextEdit" name="source">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="ppdate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="output">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Source code address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>ppDATE address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Output address:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>250</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_options">
    <attribute name="title">
     <string>Options</string>
    </attribute>
    <widget class="QRadioButton" name="only_parse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Only parsing</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="only_rv">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Only runtime verification</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="none_verbose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>None verbose monitor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="xml">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Do not remove KeY xml file</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_console">
    <attribute name="title">
     <string>Console</string>
    </attribute>
    <widget class="QTextEdit" name="console">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>561</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_run">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Abort</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
